### ISSUE TITLE
Error when running dammit 0.0.8

### ISSUE THREAD

**Issue by ctb** [camillescott/dammit#27]
```
[ ] hmmscan:longest_orfs.pep.x.Pfam-A.hmm


Error: Failed to open sequence file /nema/short.fa.transdecoder_dir/longest_orfs.pep for reading
```
Apparently it's under the dammit directory:

```
# find /nema -name longest_orfs.pep
/nema/short.fa.dammit/short.fa.transdecoder_dir/longest_orfs.pep
```

**Comment by camillescott** [comment]
Oh my, that's a problem. Will push a fix shortly.

**Comment by ctb** [comment]
I can run tests fairly quickly at this point on master - let me know when.

**Comment by camillescott** [comment]
Hmm, I can't seem to reproduce this. Are you working from a polluted output
dir?

On Fri, Nov 20, 2015 at 9:57 AM, C. Titus Brown <notifications@github.com>
wrote:

> I can run tests fairly quickly at this point on master - let me know when.
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/camillescott/dammit/issues/27#issuecomment-158474522>.
>



-- 
Camille Scott

Department of Computer Science
Lab for Data Intensive Biology
University of California, Davis

camille.scott.w@gmail.com

**Comment by ctb** [comment]
Seems to be because I'm running from another directory -- that is, if I do:

```
cd /nema
dammit annotate short.fa
```

things work fine; if, instead, I do:

```
cd /home
dammit annotate /nema/short.fa
```

I get the failure.  This happens with both 0.0.8 and master.

**Comment by camillescott** [comment]
Ahh -- I can probably fix this with a call to os.path.abspath somewhere :)

**Comment by ctb** [comment]
On Sun, Nov 22, 2015 at 08:19:06PM -0800, Camille Scott wrote:
> Ahh -- I can probably fix this with a call to os.path.abspath somewhere :)

I breathlessly await confirmation of such!

**Comment by camillescott** [comment]
Resolved in 0.0.10; closing.
